dist/metricsgraphics.css

```diff
@@ -257,6 +334,31 @@ tspan.hist-symbol {
     fill: #5c5c5c;
 }
 
+.mg-line6-legend-color {
+    color: steelblue;
+    fill: steelblue;
+}
+
+.mg-line7-legend-color {
+    color: #f673bf;
+    fill: #f673bf;
+}
+
+.mg-line8-legend-color {
+    color: #0b73b0;
+    fill: #0b73b0;
+}
+
+.mg-line9-legend-color {
+    color: #006400;
+    fill: #006400;
+}
+
+.mg-line10-legend-color {
+    color: #92514f;
+    fill: #92514f;
+}
+
```

Comment:
all the css has been included on the basis of logic only and is pretty easy to understand :smile: